### PR TITLE
Fix - Updated pie chart for top stores and added tooltip for card components

### PR DESCRIPTION
**Pull request by naresh130199**
…rd components as well

### CODE REVIEW

File: dashboard/src/components/Cards/Card/Card.jsx [hunk 1]
```diff
@@ -15,9 +15,15 @@ export const Card = ({
           <h4 className=' pr-4 text-lg md:text-3xl'>{title}</h4>
           {/* //TODO To be added in Next PR */}
           {info.length > 0 ? (
-            <h1 title={info} className='material-icons text-sm text-black/70'>
-              info
-            </h1>
+            <div className='tooltip'>
+              <h1
+                // title={info}
```
Comment (omni-srejitk) [inline dashboard/src/components/Cards/Card/Card.jsx:20]: Remove this if not needed.

File: dashboard/src/routes/Analytics/Analytics.jsx [hunk 2]
```diff
@@ -111,58 +110,57 @@ export const Analytics = () => {
     array?.forEach((a) => {
       liveStoresData?.forEach((l) => {
```
Comment (omni-srejitk) [inline dashboard/src/routes/Analytics/Analytics.jsx:111]: Use a better name for the individual object here. l gives no sense of what this is about

File: dashboard/src/routes/Analytics/Analytics.jsx [hunk 3]
```diff
@@ -111,58 +110,57 @@ export const Analytics = () => {
...
+    let valueArray = []; //array that has stores which constitute 80% of business
+    let sumOfValues = 0; //total sum of value (business money)
+    let sum80 = 0; //to push elements which constitute 80% of sumOfValues
+    topStores?.forEach((element) => {
+      sumOfValues += element[1];
+    });
+    topStores?.forEach((element) => {
+      sum80 += element[1]; //adding value to sum80
+      if (sum80 <= sumOfValues * 0.8) {
+        //if sum80 is less than 80% of total value
```
Comment (omni-srejitk) [inline dashboard/src/routes/Analytics/Analytics.jsx:137]: Again iterating that you need to clean this bit of code up. The comments, variable names everything makes it messy

**Review by omni-srejitk — CHANGES_REQUESTED** [review]
Look into the suggestions.

File: dashboard/src/components/Cards/Card/Card.jsx (hunk 1 above)
Comment (naresh130199) [reply dashboard/src/components/Cards/Card/Card.jsx:20]: done

**Review by omni-srejitk — APPROVED** [review]

**Pull request merged**